### DISCUSSION TITLE
Respository import hanging for hours

### DISCUSSION THREAD

**Discussion by lugn621** [community/community#27233] [Other Feature Feedback, Questions, & Ideas]
I'm trying to import an existing repository but it's been hanging at the " Importing commits and revision history" stage for almost a whole day. I've seen from other user's discussions of a similar issue it was suggested that it could be that there is a large queue of imports GitHub has to work through, but I'm not sure if it should take this long.

**Comment by jmansilla-2014056** [comment]
1. You can tell us with details which is the repository you are trying to import.
2. The repository manager has pending reviews, you should wait for the reviews to finish.
3. Alternative A: Try to use the command line https://docs.github.com/en/get-started/importing-your-projects-to-github/importing-source-code-to-github/importing- a-git-repository-using-the-command-line
4. Alternative B
Download GitHub Desktop for Windows
